jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/data/FeaturePreprocessor.scala

```diff
@@ -65,3 +65,32 @@ case class PrimitiveProcessor() {
   def processLong(l: Long): Long = l
   def processString(s: String): String = s
 }
+
+object PrimitiveProcessors {
+
+  def fromFunctionMaps(featuresConfig: FeaturesConfig,
+                       tfRecordType: String,
+                       fFuncs: Map[String, Float => Float],
+                       lFuncs: Map[String, Long => Long],
+                       sFuncs: Map[String, String => String]): Map[DataType, Map[String, PrimitiveProcessor]] = {
+    featuresConfig
```

Comment:
Do we want to support boolean values here for extensibility? Now that we are making this generic, we should support boolean features (they can still be passed as floats 1/0). Maybe in v2.

Reply:
DataType.BOOL

Reply:
SequenceExample in TF protobuf-land does not support those - only 32bit float, String == utf8-bytes, and 64bit long ints